--- a/containerd.service	2023-02-21 16:14:41.843267800 +0100
+++ b/containerd.service	2023-02-21 16:15:09.963315000 +0100
@@ -21,7 +21,7 @@
 #uncomment to enable the experimental sbservice (sandboxed) version of containerd/cri integration
 #Environment="ENABLE_CRI_SANDBOXES=sandboxed"
 ExecStartPre=-/sbin/modprobe overlay
-ExecStart=/usr/local/bin/containerd
+ExecStart=/usr/bin/containerd
 
 Type=notify
 Delegate=yes
